lib/interop.js

```diff
@@ -261,7 +261,7 @@ export class Interop {
             } else if (mLine.msid) {
                 // Add sources and source-groups to the existing m-line of the same media type.
                 if (mLine.sources && Array.isArray(mLine.sources)) {
-                    media[type].sources = media[type].sources.concat(mLine.sources);
+                    media[type].sources = (media[type].sources || []).concat(mLine.sources);
```

Comment:
Why is this change needed ? We are checking if `mLine.sources` is not `undefined` and that it is an `array` before trying to concat.